core-bundle/src/Command/AutomatorCommand.php

```diff
@@ -49,7 +49,7 @@ protected function configure(): void
     {
         $this
             ->setName('contao:automator')
-            ->addArgument('task', InputArgument::OPTIONAL, "The name of the task:\n  - ".implode("\n  - ", $this->getCommands()))
```

Comment:
I think this actually hurts DX because now you have no way to know which tasks there are.
Symfony introduced [lazy loading of commands](https://symfony.com/doc/current/console/commands_as_services.html#lazy-loading) for exactly these purposes. Imho that should be the way to go here?

Reply:
That would still boot the framework when listing the help information (for all commands)?

Reply:
Yeah but only once I run the command or `--help`. Not by default like it does currently. So it's not going to affect other commands.

Reply:
changed in 35be2152831ec25cb9988433bbe6d1a5b1a43559